src/platforms/stm32/rtt_if.c

```diff
@@ -128,7 +128,7 @@ uint32_t rtt_write(const char *buf, uint32_t len)
 		}
 		/* flush 64-byte packet on full-speed */
 		if (CDCACM_PACKET_SIZE == 64 && (len % CDCACM_PACKET_SIZE) == 0)
-			while(usbd_ep_write_packet(usbdev, CDCACM_UART_ENDPOINT, NULL, 0) <= 0);
+			usbd_ep_write_packet(usbdev, CDCACM_UART_ENDPOINT, NULL, 0);
```

Comment:
wonder if this should just be a while look for `< 0` - it was missing a `continue` statement anyway.

Does
```c
while (usbd_ep_write_packet(usbdev, CDCACM_UART_ENDPOINT, NULL, 0) < 0)
    continue;
```
work, or is that also broken? Asking as this removes all error recovery which might result in data loss

Reply:
ah, hmm.. that's not likely going to change so we'll take this as-is then and merge.